src/hugr/views.rs

```diff
@@ -52,14 +52,23 @@ pub trait HugrView: sealed::HugrInternals {
     where
         Self: 'a;
 
-    /// Return index of HUGR root node.
-    fn root(&self) -> Node;
+    /// Return the root node of this view.
+    #[inline]
+    fn root(&self) -> Node {
```

Comment:
This is just a way of exposing the hidden-in-HugrInternals root_node, right?
(Might one call that `root` too and then do `(self as HugrInternals).root()` here?)

Reply:
Yeah, I don't seriously suggest that, think I've got it now

Reply:
The different name is to avoid ambiguity at local call sites. Users shouldn't see the `HugrMutInternals`, so there shouldn't be any confusion.